NitroxModel/DataStructures/ThreadSafeList.cs

```diff
@@ -67,7 +67,7 @@ public ThreadSafeList(IEnumerable<T> values)
             list = new List<T>(values);
         }
 
-        public ThreadSafeList(IList<T> list, bool createCopy = true)
+        public ThreadSafeList(List<T> list, bool createCopy = true)
```

Comment:
No, there is no dedicated processor. By default it would use the IEnumerable<T> processor which is much less performant. By using a normal List<T> instead it can use its internal array to achieve better performance.

Reply:
Looking at `BinaryPack.Models.ListContainerModel` it seems BinaryPack  can handle IList